### PR TITLE
feat(components): move getAssetLogoUrl to asset-utils

### PR DESCRIPTION
**Pull request by vytick**
- move `getAssetLogoUrl` to `package/asset-utils`
- use getAssetLogoUrl to fetch contract icons from data.trezor.io

## Related Issue

Resolve ##15122

## Screenshots:

https://github.com/user-attachments/assets/88a4fb49-ebcb-4d35-a6f0-4042333560c5

### CODE REVIEW

File: suite-native/icons/src/CryptoIcon.tsx [hunk 1]
```diff
@@ -23,26 +18,40 @@ export const cryptoIconSizes = {
...
-    return tokenIcons[symbol.toLowerCase() as TokenIconName] ?? genericTokenIcon;
-};
+    const coingeckoId = getCoingeckoId(symbol);
 
-export const CryptoIcon = ({ symbol, size = 'small' }: CryptoIconProps) => {
-    const iconFile = getIconFile(symbol);
-    const sizeNumber = typeof size === 'number' ? size : cryptoIconSizes[size];
+    let sourceUrl = cryptoIcons[symbol.toLowerCase() as CryptoIconName];
+    if (coingeckoId && contractAddress) {
+        const formattedAddress = getContractAddressForNetwork(symbol, contractAddress);
```
Comment (Nodonisko) [inline suite-native/icons/src/CryptoIcon.tsx]: UseMemo

**Review by Nodonisko — CHANGES_REQUESTED** [review]
You need to modify also generate icons script it expects token folder to be present.

**Comment by vytick** [comment]
> You need to modify also generate icons script it expects token folder to be present.

I don't see any reference to tokenIcons other than in MD file

**Comment by vytick** [comment]
> You need to modify also generate icons script it expects token folder to be present.

@Nodonisko I don't see any reference to tokenIcons other than in MD file

File: suite-native/icons/src/CryptoIcon.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/icons/src/CryptoIcon.tsx]: 6f073ad407

**Review by Nodonisko — APPROVED** [review]
Yea, sorry I forget that token icons was generated manually

**Pull request merged**